src/components/MultiMatrix/MultiMatrix.vue

```diff
@@ -436,6 +436,19 @@ export default Vue.extend({
         .enter()
         .append('g')
         .attr('class', 'column')
+        .attr('transform', (d: Node) => {
+          if (d.type === 'node') {
+            return `translate(${this.orderingScale(
+              d.parentPosition,
+            )})rotate(-90)`;
+          } else {
+            return `translate(0, 0)`;
```

Comment:
This needs to be rotated
```suggestion
            return `translate(0, 0)rotate(-90)`;
```